pom.xml

```diff
@@ -160,6 +160,8 @@
         <quartz.version>2.3.1</quartz.version>
         <tinkergraph.version>3.3.3</tinkergraph.version>
         <lucene-analyzers-common.version>7.0.0</lucene-analyzers-common.version>
+        <cassandra.version>3.5.0</cassandra.version>
+        <guava.version>20.0</guava.version>
```

Comment:
This version of guava is from Oct 2016 - and quite likely introduces more security issues into our scans. Was there a reason for choosing this version? Also I think we should be reviewing new dependencies - though we have discussed cassandra - was this mandated by cassandra, or our code? cc: @mandy-chessell Current is 28.1, with some refactoring from 23.x
Merged now so we may need to raise issue in any changes we think are necessary?

Cassandra client is also from April 2016 . 3.x goes to 3.11.4 in Feb 2019, then v4 (maybe a bigger change) - again why was this version used?